package.json

```diff
@@ -58,8 +58,7 @@
     "eslint-config-okonet": "^5.0.1",
     "eslint-plugin-node": "^5.1.1",
     "husky": "^0.14.3",
-    "jest": "^21.1.0",
-    "jest-cli": "^21.1.0",
+    "jest": "^22.0.4",
```

Comment:
Since we do not use lock files, let us change this to the exact version.

Reply:
It would be a synthetic lockdown, as all transitive dependencies will still be semver ranges. I can still do it, of course, but it won't really make any difference

Reply:
I didn't realise that. I would vote for a `yarn.lock` but we probably won't have a consensus on that. @luftywiranda13, @okonet what do you suggest.

Reply:
I'm okay with committing lockfile to this project. I prefer `yarn.lock` too

Reply:
I'm okay with both `package-lock.json` or `yarn.lock` but then we need to remove the "removal" step. @luftywiranda13 can you do that please?

Reply:
@okonet sure https://github.com/okonet/lint-staged/pull/367#discussion_r159025037 👍